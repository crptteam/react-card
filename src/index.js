import Card from "./components/Card";
import CardLine from "./styled/CardLine";

export { Card, CardLine };

export default Card;
