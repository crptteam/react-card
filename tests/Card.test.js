import React from "react";
import { Card } from "../src/";

describe("Card", () => {
  it("Should renders without problems", () => {
    const wrapper = shallow(<Card>This is card.</Card>);

    expect(wrapper).toMatchSnapshot();
  });
});
