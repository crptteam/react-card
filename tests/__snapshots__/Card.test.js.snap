// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Should renders without problems 1`] = `
<CardWrap
  theme={
    Object {
      "background": "#FFFFFF",
      "paddingBottom": "40px",
      "paddingLeft": "32px",
      "paddingRight": "32px",
      "paddingTop": "32px",
    }
  }
>
  This is card.
</CardWrap>
`;
